tools/cldr-apps/src/main/webapp/js/new/cldrLoad.js

```diff
@@ -1077,36 +1077,6 @@ const cldrLoad = (function () {
     );
   }
 
-  function flipToOtherDiv(div) {
-    flipper.flipTo(pages.other, div);
```

Comment:
some code was duplicated, probably I did copy-paste when I intended cut-paste -- webpack noticed this too